product_docs/docs/epas/15/managing_performance/04_dynamic_runtime_instrumentation_tools_architecture_DRITA/taking_a_snapshot.mdx

```diff
@@ -19,9 +19,7 @@ EDB Postgres Advanced Server's `postgresql.conf` file includes a configuration p
 SET timed_statistics = TRUE
```

Comment:
- "Connect to the server with the EDB-PSQL client, and invoke the command:" is duplicated. 
- The indentation of the code blocks is off.
- I wonder if much of this should be turned into a procedure

Reply:
Hi, @drothery-edb! I am going to merge this PR, but will address these comments in the other branch I am working in now, if that's OK.